inference-engine/include/gpu/gpu_ocl_wrapper.hpp

```diff
@@ -13,17 +13,34 @@
 /**
 * @brief Definitions required by Khronos headers
 */
-#define CL_HPP_ENABLE_EXCEPTIONS
-#define CL_HPP_MINIMUM_OPENCL_VERSION 120
-#define CL_HPP_TARGET_OPENCL_VERSION 120
 
-#if defined __GNUC__
+#ifndef CL_HPP_ENABLE_EXCEPTIONS
+# define CL_HPP_ENABLE_EXCEPTIONS
+#endif
+
+#ifdef CL_HPP_MINIMUM_OPENCL_VERSION
+# if CL_HPP_MINIMUM_OPENCL_VERSION <= 120
+#  error "CL_HPP_MINIMUM_OPENCL_VERSION must be >= 120"
+# endif
+#else
+# define CL_HPP_MINIMUM_OPENCL_VERSION 120
+#endif
+
+#ifdef CL_HPP_TARGET_OPENCL_VERSION
+# if CL_HPP_TARGET_OPENCL_VERSION <= 120
+#  error "CL_HPP_TARGET_OPENCL_VERSION must be >= 120"
+# endif
+#else
+# define CL_HPP_TARGET_OPENCL_VERSION 120
+#endif
```

Comment:
As I see onednn plugin requires 200 version, while our headers 120 version. Are you OK with such changes in header?

Reply:
It is ok - onednn plugin is being also switched to 120. Thank you.

Reply:
lgtm as well